distribution/lib/Standard/Examples/0.0.0-dev/src/Main.enso

```diff
@@ -1,6 +1,7 @@
 from Standard.Base import all
 
 import Standard.Base.Data.Text.Regex.Match.Match
+import Standard.Base.Data.Text.Regex.Regex
```

Comment:
Not needed as in `from Standard.Base import all`

Reply:
Done